awx/ui_next/src/screens/Project/ProjectList/ProjectList.jsx

```diff
@@ -212,13 +251,14 @@ function ProjectList() {
             )}
             renderRow={(project, index) => (
               <ProjectListItem
-                fetchProjects={fetchProjects}
+                fetchUpdatedProjects={fetchUpdatedProjects}
```

Comment:
```suggestion
                fetchProject={fetchUpdatedProjects}
```

This prop is still called `fetchProject` [in the list item](https://github.com/ansible/awx/blob/devel/awx/ui_next/src/screens/Project/ProjectList/ProjectListItem.jsx#L45) so to avoid errors we'll need to keep using the `fetchProjects` prop or also update it in the list item.

![fetchProjectProp](https://user-images.githubusercontent.com/9753817/120503337-3746e780-c391-11eb-8cb0-ccae7db4d1ae.png)

Reply:
Oof - good catch